sdk/go/common/util/cmdutil/args.go

```diff
@@ -28,10 +28,8 @@ func ArgsFunc(argsValidator cobra.PositionalArgs) cobra.PositionalArgs {
 		err := argsValidator(cmd, args)
 		if err != nil {
 			contract.IgnoreError(cmd.Help())
```

Comment:
Very nitty nit: there is no reason to keep `contract.IgnoreError`
```suggestion
			return errors.Join(cmd.Help(), err)
```